cypress/support/commands.ts

```diff
@@ -113,10 +113,12 @@ Cypress.Commands.add('signup', ({ firstName, lastName, password, url }) => {
 	});
 });
 
-Cypress.Commands.add('setup', ({ email, firstName, lastName, password }) => {
+Cypress.Commands.add('setup', ({ email, firstName, lastName, password }, skipIntercept = false) => {
```

Comment:
why not always wait?

Reply:
Because of `should be able to setup instance and migrate workflows and credentials` spec above. In which we also need to click away messageBox before continuing with the setup and awaiting the intercept:
```js
it('should be able to setup instance and migrate workflows and credentials', () => {
		cy.setup({ email, firstName, lastName, password }, true);

		messageBox.getters.content().should('contain.text', '1 existing workflow and 1 credential');

		messageBox.actions.confirm();
		cy.wait('@setupRequest');
	```